pkg/feature/builder.go

```diff
@@ -204,11 +204,15 @@ func (fb *featureBuilder) withDefaultClient() error {
 	return nil
 }
 
-// ManifestSource sets the root file system (fsys) from which manifest paths are loaded
-// If ManifestSource is not called in the builder chain, the default source will be the embeddedFiles.
-func (fb *featureBuilder) ManifestSource(fsys fs.FS) *featureBuilder {
+// Used to enforce that Manifests() is called after ManifestSource() in the chain.
+type featureBuilderWithManifestSource struct {
+	*featureBuilder
+}
+
+// ManifestSource sets the root file system (fsys) from which manifest paths are loaded.
+func (fb *featureBuilder) ManifestSource(fsys fs.FS) *featureBuilderWithManifestSource {
```

Comment:
nit, but maybe worth moving it higher up in the file so it's aligned with the chain.

Reply:
done, moved both up